Cargo.lock

```diff
@@ -1655,4 +1655,4 @@ checksum = "79af3189e6b0484c9fd54208f8eeb8818cadee00ec81438b67a64c8e6f2f3694"
 dependencies = [
  "bit-vec",
  "num-bigint",
-]
+]
```

Comment:
@irajtaghlidi is this change intentional?

Reply:
Fixed.